### PR TITLE
new flockmtl release v0.2.1

### PR DESCRIPTION
**Pull request by queryproc**
Update the description to point to the latest release: https://github.com/dsg-polymtl/flockmtl/releases/tag/v0.2.1

### CODE REVIEW

File: extensions/flockmtl/description.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 extension:
   name: flockmtl
-  description: DuckDB LLM & RAG Extension
+  description: DuckDB LLM & RAG Extension [(Docs)](https://dsg-polymtl.github.io/flockmtl/docs).
   version: 0.2.0
```
Comment (carlopi) [inline extensions/flockmtl/description.yml:4]:
```suggestion
  version: 0.2.1
```

Comment (carlopi) [reply extensions/flockmtl/description.yml:4]: I think this should be 0.2.1, I will assume so and commit with this, but please @queryproc check on this

**Comment by carlopi** [comment]
Thanks, left a comment, I think version field should be 0.2.1, but good to go anyway.

File: extensions/flockmtl/description.yml (hunk 1 above)
Comment (queryproc) [reply extensions/flockmtl/description.yml:4]: Yeah, you are right. I will update the version with the next release.

**Pull request merged**